src/dndbeyond/base/utils.js

```diff
@@ -218,6 +218,9 @@ async function buildAttackRoll(character, attack_source, name, description, prop
             const choice = await queryDamageTypeFromArray(roll_properties.name, damages, damage_types, ["Acid", "Cold", "Fire", "Lightning", "Poison", "Thunder"]);
             if (choice === null) return null; // Query was cancelled;
         } else if (roll_properties.name === "Sorcerous Burst") {
+            // fix dndbeyond dmg issue
+            damages = damages.map(m => damages[0]); // set everything to the correct acid dmg
```

Comment:
Hummm.. are we sure this happens very very early on? Before any other damages are added, like sneak attack/rage/etc... (not that they'd apply for a spell, but you know what I mean).

Reply:
not for this spell, other spells like shadow blade sneak does apply